### PR TITLE
setup-and-movil-version-skeleton

### PR DESCRIPTION
**Pull request by MarcoDiaz2000**
**[Project 1: Setup and mobile version skeleton]** 

The first milestone in the process of creating my portfolio website:

Set up a new repository and prepare it for development using best practices (e.g. linters).
Build the first 2 sections of the mobile website.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <script src="https://kit.fontawesome.com/84fe87cdc9.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="mylogo">My Logo</div>
+        <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
+        </div>
```
Comment (BregornOriginal) [inline index.html]: Good job creating a `header` that will contain the top information on your website

- Kindly, follow semantic rules on HTML, wrap inside a `nav` tag your logo and hamburger icon 💯

File: index.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    </header>
+    <section class="hero">
+        <h1 class="title">I’m Alexandra. Glad to see you!</h1>
+        <p class="text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <div class="socialtitle">LET’S CONNECT</div>
+        <div>
+            <ul class="socialicon">
```
Comment (BregornOriginal) [inline index.html:28]: - Kindly align your social icons with the title as it on Figma design

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @MarcoDiaz2000  

## Good job so far!
## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good pull request description
✅ Excellent work 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -13,16 +13,18 @@
...
 <body>
     <header>
-        <div class="mylogo">My Logo</div>
-        <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
-        </div>
+        
+            <div class="mylogo">My Logo</div>
+            <span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
+            </div>
+
```
Comment (ichala) [inline index.html]: - [ ] as mentioned by the previous reviewer kindly wrap your logo and hamburger button inside a nav tag, example : 
```HTML
<nav>
<div class="mylogo">My Logo</div>
<span class="mobilmenu"><i class="fa-solid fa-bars"></i></span>       
</nav>
```

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 


 ## Required Changes ♻️

- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @MarcoDiaz2000, 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linter error.
:heavy_check_mark: Descriptive readme file.
:heavy_check_mark: Well-implemented suggestions.

### Optional suggestions :arrow_down_small: 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [x] The commit message should be written in the imperative tense and the first letter should also be capitalized. `Ex: "Fix bug" instead of "fix bug".` [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

- [x]  Currently, your `README` file is not professional enough. Please update your `README` file. You can use [this template ](https://github.com/microverseinc/readme-template)for the `README` but remember to customize it to your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**